src/utils/clack/index.ts

```diff
@@ -1651,3 +1654,193 @@ export async function askShouldAddPackageOverride(
+      value: 'proceed',
+    },
+  );
+
+  const whatToDo = await abortIfCancelled(
+    clack.select({
+      message: `We couldn't find build artifacts at "${relativeArtifactPath}". What would you like to do?`,
+      options,
+      initialValue: buildCommand ? 'run-build' : 'manual',
+    }),
+  );
+
+  if (whatToDo === 'proceed') {
+    return {
+      validPath: true,
+    };
+  }
+
+  if (whatToDo === 'run-build' && buildCommand) {
+    const ranBuildAndCheckArtifacts = await runBuildAndCheckArtifacts(
+      buildCommand,
+      relativeArtifactPath,
+    );
+
+    if (ranBuildAndCheckArtifacts.validPath === false) {
+      return await promptWhatToDo({ relativeArtifactPath });
+    }
+
+    return ranBuildAndCheckArtifacts;
+  }
+
+  return {
+    validPath: false,
+  };
+}
+
+async function getPossibleBuildFolders(): Promise<string[]> {
+  const commonBuildFolders = ['build', 'dist', 'out', '.next'];
+
+  const existingFolders: string[] = [];
+
+  for (const folder of commonBuildFolders) {
+    try {
+      await fs.promises.access(path.join(process.cwd(), folder));
+      existingFolders.push(folder);
+    } catch {
+      // Folder doesn't exist, continue
+    }
+  }
+
+  return existingFolders;
+}
+
+async function promptForAlternativeArtifactPath(): Promise<string | undefined> {
+  const folders = await getPossibleBuildFolders();
+
+  if (!folders.length) {
+    return undefined;
+  }
+
+  if (folders.length === 1) {
+    const [onlyFolder] = folders;
+
+    const confirmed = await abortIfCancelled(
+      clack.select({
+        message: `Detected one possible build folder. Use "${onlyFolder}" as your build artifacts folder?`,
+        options: [
+          {
+            value: true,
+            label: 'Yes',
+          },
+          { value: false, label: 'No' },
+        ],
+      }),
+    );
+
+    return confirmed ? onlyFolder : undefined;
+  }
+
+  const selected = await abortIfCancelled(
+    clack.select({
+      message: `Detected multiple possible build folders. Is one of these your build artifacts folder?`,
+      options: [
+        ...folders.map((f) => ({ value: f, label: f })),
+        { value: false, label: 'No' },
+      ],
+    }),
+  );
+
+  if (selected) {
+    return String(selected);
+  }
+
+  return undefined;
+}
+
+async function runBuildCommand(buildCommand: string): Promise<boolean> {
+  const packageManager = await getPackageManager(NPM);
+  const command = `${packageManager.runScriptCommand} ${buildCommand}`;
+  const spinner = clack.spinner();
+
+  spinner.start(`Running ${chalk.cyan(command)}...`);
+  try {
+    execSync(command, {
+      stdio: 'inherit',
+      cwd: process.cwd(),
+    });
+    spinner.stop('Build finished running.');
+    return true;
+  } catch (error) {
+    spinner.stop(
+      `Build failed: ${error instanceof Error ? error.message : String(error)}`,
+    );
+    clack.log.error(
+      'Failed to run build command. Please run it manually and try again.',
```

Comment:
l: 

```suggestion
      'Failed to run your build command. Please run it manually and try again.',
```

Might make it a bit clearer that it's their build command we failed to run, not the wizard's?